### DISCUSSION TITLE
Preview not triggering on AppleTV

### DISCUSSION THREAD

**Discussion by GenericDownloader** [dgreif/ring#997] [Q&A]
Since the update that removed the option to simulate a doorbell press, the preview for motion isn’t showing on AppleTV. The settings in AppleTV are set to show if there’s motion, but it doesn’t work. It’s like Homebridge isn’t sending something to the AppleTV to say there’s motion. 

Other cameras that are native HKSV work fine, it’s just Ring via homebridge since the update about a month ago. 

Any ideas?

**Comment by bgwallace** [comment]
I'm having the same issue GenericDownloader.  Has anyone found a solution?

**Comment by dgreif** [comment]
Are you getting motion notifications on your phone? It's possible that the motion events just aren't being picked up at all, which a few users have experienced.  I've got some ideas on how to tackle that problem, but haven't had time to implement it yet

**Reply by GenericDownloader** [reply]
Yep I'm getting the notifications on my phone and other devices. Just doesn't pop up on the Apple TV.

**Reply by dgreif** [reply]
Weird.  Does live streaming workin on your AppleTV for this camera? If you just try to access the live stream directly

**Reply by GenericDownloader** [reply]
Yep live streaming works. Sorry these aren't useful answers!

I've played around with the motion settings in the Ring app just to make them a little more sensitive to make sure alerts should be coming through.

**Reply by ggogins** [reply]
I'm having the exact same issue.  Live stream works fine, just no motion pop up on Apple TV.  Worked great prior to 11

**Comment by bgwallace** [comment]
I'm not sure why but at some point this started working for me.  I could be wrong but I don't believe I made any changes between the time when it wasn't working and when it started to work.
